renovate.json

```diff
@@ -3,5 +3,25 @@
 	"extends": [
 		"github>jellyfin/.github//renovate-presets/gradle",
 		":semanticCommitsDisabled"
-	]
+	],
+	"group": {
+		"packageRules": [
+			{
+				"groupName": "Kotlin",
+				"matchPackageNames": [
+					"com.google.devtools.ksp"
+				],
+				"matchPackagePrefixes": [
+					"org.jetbrains.kotlin",
+					"org.jetbrains.kotlinx:kotlinx-coroutines"
+				]
+			},
+			{
+				"groupName": "detekt",
+				"matchPackagePrefixes": [
+					"io.gitlab.arturbosch.detekt"
+				]
+			}
+		]
+	}
```

Comment:
These should go to:
https://github.com/jellyfin/.github/blob/master/renovate-presets/gradle.json

Reply:
I literally didn't find the GitHub repo earlier...

But are we sure we want to enable that for all repos?

Reply:
The preset is used for jellyfin-android, jellyfin-androidtv and jellyfin-sdk-kotlin so yes.

Reply:
Alright. Superseded by jellyfin/.github#11 then.

Reply:
I'll use this PR to fix the formatting then, I saw the `.github` repo uses 2 spaces too.